src/utils.js

```diff
@@ -1,127 +1,145 @@
 // A simple generator function to provide temporary IDs for generated resources.
-function* simpleGenerator() {
-  let n = 1;
-  while (true)
-    yield n++;
+// TODO: why not use UUIDs?
```

Comment:
For testing.  Much easier to write tests when the generated resources are given short and sequential IDs vs. random UUIDs (even if you seed your RNG).